### DISCUSSION TITLE
Quill 2.0 Bullet List adds <ol> tags

### DISCUSSION THREAD

**Discussion by AlexBindevald** [slab/quill#4189] [Q&A]
Hello, 
I am creating an component in Vue 3 with Primevue. There is component called Editor which uses quill. When I install latest quill version then the bullet list option in toolbar adds ol tags instead of ul tags. In 1.3.7 version it works correctly.  What is the workaround since the 1.3.7 is not safe to use.

**Comment by luin** [comment]
You can use https://quilljs.com/docs/api#getsemantichtml to get the HTML representation of the editor instead of just getting the `innerHTML` as `innerHTML` contains some edit-only DOM nodes, e.g. toolbars.
